src/Psy/TabCompletion/Matcher/AbstractMatcher.php

```diff
@@ -112,7 +112,11 @@ protected function getNamespaceAndClass($tokens)
      */
     public static function startsWith($prefix, $word)
     {
-        return preg_match(sprintf('#^%s#', $prefix), $word);
+        if (empty($prefix)) {
```

Comment:
you can't use `empty` with strings in PHP.

```php
empty("0")
```

ಠ_ಠ

Reply:
you shouldn't program in two different programming languages on two monitors at the same time.